tsconfig.es.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "target": "ES2019",
+        "module": "es2015",
+        "outDir": "lib/es",
+        "moduleResolution": "node"
```

Comment:
Does `moduleResolution` have to be set here?

Reply:
If you don't specify node it will classic if you change module to es2015.
Described here: https://www.typescriptlang.org/docs/handbook/module-resolution.html#module-resolution-strategies